src/Microsoft.TestPlatform.Extensions.HtmlLogger/HtmlLogger.cs

```diff
@@ -336,7 +336,6 @@ private void PopulateHtmlFile()
                 EqtTrace.Error("HtmlLogger : Failed to populate html file. Exception : {0}",
                     ex.ToString());
                 ConsoleOutput.Instance.Error(false, string.Concat(HtmlResource.HtmlLoggerError), ex.Message);
-                return;
```

Comment:
Any reason for this removal? It changes the current behavior.

Reply:
yes you are right. reverting.